app/elements/list-links.mjs

```diff
@@ -0,0 +1,30 @@
+export default function ListLinks({ html, state = {} }) {
+  const { attrs } = state
+  const { links } = attrs
+  const { linkedin, twitter, url, github } = { ...links }
+  return html`
+    <style>
+      ul {
+        list-style: none;
+        margin: 8px 0 0 0;
+        padding: 0;
+      }
+    </style>
+    <ul>
+      ${linkedin
+        ? `<li class="links-list"><i class="fab fa-linkedin" style="width: 20px;"></i><a target="_blank" href="https://www.linkedin.com/in/${linkedin}">${linkedin}</a></li>`
+        : ''}
+      ${twitter
+        ? `<li class="links-list"><i class="fab fa-x-twitter" style="width: 20px;"></i><a target="_blank" href="https://twitter.com/${twitter}">@${twitter}</a></li>`
```

Comment:
Don't really care if we use the X or the bird icon here, happy to defer if anyone has a preference